lms/static/sass/shared/_footer.scss

```diff
@@ -190,6 +190,11 @@
       }
     }
   }
+
+  @media print {
+    border: none;
+    padding: 10px 10px 0 10px;
```

Comment:
Part of this was trying to deal more globally with openedx sites using the old footer vs the new footer. It may well have been extraneous.